views/dashboard/index.html

```diff
@@ -1,9 +1,13 @@
-<br/>
-<br/>
-<form action="{{ urlfor `ScraperController.Create` }}" method="post">
-  <label for="scrapper_keyword">keyword: </label>
-  <input type="text" id="scrapper_keyword" name="keyword" placeholder="Search keyword here..." required>
-  <br/>
-  <br/>
-  <button type="submit" class="button button--primary">Search</button>
-</form>
+<div class="search-keyword">
+  <form class="form" action="{{ urlfor `ScraperController.Create` }}" method="post">
+    <div class="form__group">
+      <input class="form__input" type="text" name="keyword" placeholder="Search keywords here..." required>
```

Comment:
@Lahphim Should we use the file field here so the user can upload the CSV file as the requirement mentioned `Authenticated users can upload a CSV file of keywords. This upload file can be of any size from 1 to 1,000 keywords.`?

Reply:
@carryall I have another tickets for CSV also here 
![Screen Shot 2564-02-25 at 16 59 12](https://user-images.githubusercontent.com/749672/109138795-567c1a80-778d-11eb-961f-fb8c7214a6b3.png)

This ticket will be just only simple input for manual search 😅
This won't including in the IC requirement, I create this ticket by mistake. 😵
But I think it's ok to have it for proving something 📃